docs/topics/ref_application-development-resources.adoc

```diff
@@ -1,7 +1,12 @@
 [id='application-development-resources']
 = Application development resources
 
-For additional information on application development with OpenShift see:
+For additional information about application development with OpenShift, see:
+
+* link:https://learn.openshift.com/[OpenShift Interactive Learning Portal^]
+* link:https://developers.redhat.com/products/rhoar/overview/[Red Hat OpenShift Application Runtimes Overview^]
+
+To reduce network load and shorten the build time of your application, set up a Nexus mirror for Maven on your {OpenShiftLocal}:
```

Comment:
I'm not sure myself what `OpenShiftLocal` resolves to at this point, but I guess it's something like "Minishift". However, this is useful in all environments.

Reply:
`Minishift or CDK`. Should I change that to OpenShift in general?